### DISCUSSION TITLE
api gateway version 2 support

### DISCUSSION THREAD

**Discussion by srikanthcs** [cloud-custodian/cloud-custodian#7405] [General]
Does custodian support policies for apigatewayv2 (http and websocket api) ?

The docs have filters/actions for rest-api resource, and rest-api resource doesn't work on http api.
Can I raise a PR supporting apigatewayv2 ?

**Comment by ajkerrigan** [comment]
There's no apigatewayv2 resource today, but PRs are welcome! The [contributor docs](https://cloudcustodian.io/docs/contribute.html) should help you get started, but feel free to ask questions here or in the [gitter channel](https://gitter.im/cloud-custodian/cloud-custodian).

There is also more AWS-specific contributor information coming via #6616. It may be a useful reference even though it hasn't landed in the official documentation yet. Thank you!
